Testscripts/Linux/perf_lkp.sh

```diff
@@ -0,0 +1,186 @@
+            install_package "${deb_packages[@]}" >> $LKP_OUTPUT 2>&1
+            ;;
+        centos_7 | redhat_7)
+            rpm_packages=(make git curl psmisc)
+            install_epel
+            LogMsg "Dependency package names： ${rpm_packages[@]}"
+            install_package "${rpm_packages[@]}" >> $LKP_OUTPUT 2>&1
+            gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+            curl -sSL https://get.rvm.io | bash -s stable
+            if [ $? -ne 0 ]; then
+                LogErr "Failed to curl rvm"
+                SetTestStateAborted
+                exit 1
+            fi
+            source /etc/profile.d/rvm.sh
+            rvm requirements
+            rvm install ruby-2.3.0
+            rvm use ruby-2.3.0 default
+            PATH="$PATH:/usr/local/bin"
+            ;;
+        *)
+            LogErr "Unsupported distro: $DISTRO"
+            return 1
+            ;;
+    esac
+}
+
+function run_tests {
+    LogMsg "git clone LKP..."
+    if [[ $DISTRO =~ "redhat" ]]; then
+        work_dir=$(df -h | grep "/dev/sdb1" | awk -F " " '{ print $6 }')
+        cd "$work_dir"
+    fi
+
+    git clone https://github.com/intel/lkp-tests.git
+    cd lkp-tests
+    make install
+    if [[ $? -ne 0 ]];then
+        LogErr "LKP make failed"
+        SetTestStateAborted
+        exit 1
+    fi
+
+    LogMsg "install LKP..."
+    yes | lkp install >> $LKP_OUTPUT 2>&1
+    if [[ $? -ne 0 ]];then
+        LogErr "LKP install failed"
+        SetTestStateAborted
+        exit 1
+    fi
+
+    LogMsg "split job..."
+    for test in ${LKP_TESTS[@]}; do
+        lkp split-job jobs/$test >> $LKP_OUTPUT 2>&1
+    done
+
+    LogMsg "install and run job..."
+    for testName in ${TEST_NAME_SET[@]}; do
+        lkp install $(ls $testName-* | head -n1) >> $LKP_OUTPUT 2>&1
+        if [[ $TEST_NAME =~ "aim9" ]]; then
+            all_jobs=$(ls aim9-all-*.yaml)
+        elif [[ $TEST_NAME =~ "unixbench" ]]; then
+            all_jobs=$(ls unixbench-1-*.yaml)
+        else
+            all_jobs=$(ls | grep $testName)
+        fi
+        for job in $all_jobs; do
+            LogMsg "run job $job..."
+            lkp run $job >> $LKP_OUTPUT 2>&1
+            if [[ $? -ne 0 ]];then
+                LogErr "LKP $job test failed"
+            fi
+        done
+        results="$(lkp result $testName)"
+
+        LogMsg "collect results..."
+        for result in $results; do
+            job_name=$(echo $result | awk -F "/" '{ print $5 }')
+            LKP_TEST_LOG="$LKP_LOG_DIR/$testName/$job_name"
+            mkdir -p "$LKP_TEST_LOG"
+            if [ -d "$result" ]; then 
+                LogMsg "copy result log to $LKP_TEST_LOG..."
+                cp $result* $LKP_TEST_LOG
+            else
+                LogErr "$result doesn't exist"
+            fi
+        done
+    done
+
+    cd $HOMEDIR
+    tar -cvf "lkp_log.tar" lkp_log
```

Comment:
consistently use $LKP_LOG_DIR? does it work?

Reply:
No. If so, the tar file will have "/home/lisa" directory. I will use $$LKP_LOG instead of "lkp_log".